src/map.h

```diff
@@ -20,6 +20,7 @@
 #include "item_stack.h"
 #include "active_item_cache.h"
 #include "string_id.h"
+#include "veh_type.h"
```

Comment:
I'm not sure I follow -- @kevingranade said to not use `vgroup_id` at all, but to just use it as `string_id<vehicle_group>`. You'd presumably still need the `vehicle_group` forward declaration, but it can still just be a forward declaration (i.e. "there exists somewhere a class named vehicle_group").